rules/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector.php

```diff
@@ -151,19 +150,27 @@ private function processToClassConstFetch(
 
                 $constName = $attributeKeyToClassConstFetch->getValuesToConstantsMap()[$value] ?? null;
                 if ($constName === null) {
-                    continue;
+                    return false;
```

Comment:
it looks fishy, better to continue, and set `$hasChanged = true;` on last when `$arg->value` changed.

Reply:
This rector is already at the limit of cognitive load, after I added the condition it was busted.
If required I could refactor it, so the return happens in a not looped context. But I wanted to keep the general changes low so the change is easier to spot.
Since the rector works on a single attribute property at a time, it's safe to return early.
The name is checked before and continues the loop when the name does not match.

Reply:
The best is `continue 2` on next parent loop, but for safest, the existing `continue` is the way to go to avoid regression for multi config and not found on the first lookup,  we learned the hard way by return too early on loop on configuration rector rule.

You can move to separate private method if you got too many logic.